libs/ebpfnetsh/maps.cpp

```diff
@@ -152,6 +152,9 @@ unsigned long
 handle_ebpf_unpin_map(
     LPCWSTR machine, LPWSTR* argv, DWORD current_index, DWORD argc, DWORD flags, LPCVOID data, BOOL* done)
 {
+    //
+    // unpin map from a specific pin path
```

Comment:
Move up above line 149, and follow style conventions (start with capitalized word, end with punctuation, no leading or trailing empty comment lines.